hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/FieldOperations.java

```diff
@@ -634,4 +868,31 @@ public static FieldTypeBasedOperations fieldOperations(FieldType fieldType) {
         FieldOperations fieldOperations = ALL[fieldType.getId()];
         return fieldOperations.operations;
     }
+
+    /**
+     * Writes arrays as json formatted.
+     * Array itself and its items are allowed to be null.
+     */
+    private static <T> void writeArrayJsonFormatted(StringBuilder stringBuilder, T[] objects,
+                                                    BiConsumer<StringBuilder, T> itemWriter) {
+        if (objects == null) {
+            stringBuilder.append("null");
+            return;
+        }
+        stringBuilder.append("[");
+        int objectSize = objects.length;
+        int j = 0;
+        for (T object : objects) {
+            j++;
+            if (object == null) {
+                stringBuilder.append("null");
+            } else {
+                itemWriter.accept(stringBuilder, object);
+            }
+            if (j != objectSize) {
+                stringBuilder.append(",");
```

Comment:
```suggestion
                stringBuilder.append(", ");
```
Minor: To make it consistent with others